examples/CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
 # Function to generate an example project with extra libraries
 function (example_project_extra project_name source_code extra_libs)
     message(VERBOSE "Adding new example project ${project_name}, sources: ${source_code}, extra_libs: ${extra_libs}")
-    file (GLOB example_project_sources ${source_code} "common/*.h" "common/*.cc")
+    # file (GLOB example_project_sources ${source_code} "common/*.h" "common/*.cc")
```

Comment:
Please drop the commented code.
Also, I would suggest to get rid of this generator function in favor of dedicated makefile per project (not in this PR).